megamek/src/megamek/common/Aero.java

```diff
@@ -897,6 +927,32 @@ public void newRound(int roundNumber) {
 
         // update velocity
         setCurrentVelocity(getNextVelocity());
+        
+        //If we're an aero on an atmospheric map, remove space-only sensors from the list of options
+        //Right now this is 'permanent' but I think it should be here under newRound for when we
+        //integrate moving between space/atmospheric maps
+        if (!game.getBoard().inSpace()) {
```

Comment:
This accounts for moving from space to atmosphere, but does not provide a way to restore sensors when moving the other direction.